bin/adam-pyspark

```diff
@@ -23,10 +23,6 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 ADAM_JARS=$("$ADAM_REPO"/bin/compute-adam-jars.sh)
 
-if [ ! -z "$ADDL_JARS" ]; then
-  ADAM_JARS="$ADAM_JARS","$ADDL_JARS"
-fi
```

Comment:
That is OK by me, but it is kind of nice to set options via envars (e.g., if you're submitting jobs via YARN and putting all your config on the command line, your command line can get quite long quite quick). What is the impetus for moving away from envars?